exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -53,6 +54,7 @@ export function rcOptionsTypes () {
       'npm-path',
       'use-node-version',
     ], allTypes),
+    'max-parallel': Number,
```

Comment:
why is it needed if there is already a `workspace-concurrency` setting?

Reply:
https://pnpm.io/cli/recursive#--workspace-concurrency

> Set the maximum number of tasks to run simultaneously. For unlimited concurrency use Infinity.

Certainly, s reading this description, `—max-parallel` is not needed. I fixed in e25b5c1 :+1: I misunderstood `—workspace-concurrency` option is just for `recursive run` and `run` with `—filter` option command.

By the above commit, pnpm runs selected scripts parallel with the number of limit specified with `workspace-concurrency`. In running scripts specified with both of a wildcard selector and `—filter` workspace selector, pnpm flattens specified script commands and runs these from top to bottom.

eg)

In the below package,

- `a` workspace:
  - `build:1` script
  - `build:2` script
  - `build:3` script
- `b` workspace:
  - `build:1` script
  - `build:2` script
  - `build:3` script

and run `pnpm --filter '*' run 'build:*'`, pnpm flattens commands to execute into [ `a` workspace's `build:1` script, `a` workspace's `build:2` script, `a` workspace's `build3` script, `b` workspace's `build:1` script, `b` workspace's `build:2` script, `b` workspace's `build:3` script ] and execute these from top to bottom.

Reply:
[fdedd17](https://github.com/pnpm/pnpm/pull/5871/commits/fdedd17b1f288e9c07e89536caa71c1d78c67007)

Additionally, I make `pnpm run` command without recursive execute specified scripts parallel with the number of limit `4` as a default behavior, because `pnpm run` command with recursive execute scripts parallel with the number of limit `4` as a default behavior and these should be the same.

So `pnpm run` with a wildcard script selector executes specified script parallel as a default behavior, so I implemented `--sequential` shorthand option. This shorthand option will specify `--workspace-concurrency=1`